.gitignore

```diff
@@ -7,3 +7,5 @@ coverage
 .settings
 composer.lock
 docs-api
+/.idea
+/composer.phar
```

Comment:
Please move these to you [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)